assets/src/components/appStateWrapper.tsx

```diff
@@ -1,21 +1,26 @@
 import React from "react"
 import { NotificationsProvider } from "../contexts/notificationsContext"
+import RoutesContext from "../contexts/routesContext"
 import { SocketProvider } from "../contexts/socketContext"
 import { StateDispatchProvider } from "../contexts/stateDispatchContext"
 import usePersistedStateReducer from "../hooks/usePersistedStateReducer"
+import useRoutes from "../hooks/useRoutes"
 import useSocket from "../hooks/useSocket"
 import App from "./app"
 
 const AppStateWrapper = (): JSX.Element => {
   const [state, dispatch] = usePersistedStateReducer()
   const socketStatus = useSocket()
+  const routes = useRoutes()
 
   return (
     <StateDispatchProvider state={state} dispatch={dispatch}>
       <SocketProvider socketStatus={socketStatus}>
-        <NotificationsProvider>
-          <App />
-        </NotificationsProvider>
+        <RoutesContext.Provider value={routes}>
```

Comment:
@skyqrose for consistency let's define a `RoutesProvider` component in `routes_context.tsx`